mwdb/web/src/components/ShowObject/Views/SharesBox.js

```diff
@@ -239,27 +280,49 @@ function SharesBox() {
     }, [getShares]);
 
     return (
-        <div className="card card-default">
-            <ConfirmationModal
-                isOpen={isModalOpen}
-                onRequestClose={() => setIsModalOpen(false)}
-                message={`Share the sample and all its descendants with ${shareReceiver}?`}
-                onConfirm={() => doShare(shareReceiver)}
-                confirmText="Share"
-                buttonStyle="bg-success"
-            />
+        <div>
+            <div className="card card-default">
+                <ConfirmationModal
+                    isOpen={isModalOpen}
+                    onRequestClose={() => setIsModalOpen(false)}
+                    message={`Share the sample and all its descendants with ${shareReceiver}?`}
+                    onConfirm={() => doShare(shareReceiver)}
+                    confirmText="Share"
+                    buttonStyle="bg-success"
+                />
 
-            <div className="card-header">
-                <div className="media">
-                    <div className="align-self-center media-body">Shares</div>
-                    <SharingStatusIcon shares={shares} />
+                <div className="card-header">
+                    <div className="media">
+                        <div className="align-self-center media-body">
+                            Shares
+                        </div>
+                        <SharingStatusIcon shares={shares} />
+                    </div>
                 </div>
+                <SharesList
+                    shares={shares}
+                    groups={groups}
+                    handleShare={!api.remote ? handleShare : null}
+                    currentFile={context.object.sha256}
+                    direct={true}
+                />
+            </div>
+            <div className="card card-default">
+                <div className="card-header">
+                    <div className="media">
+                        <div className="align-self-center media-body">
+                            Related shares
```

Comment:
Small change but should be more informative about "how it's related"
```suggestion
                            Inherited shares
```